### ISSUE TITLE
System.getConfig doesn't return all config

### ISSUE THREAD

**Issue by casperlamboo** [systemjs/systemjs#1344]
I tried to load the config from `System.getConfig()`. But it's missing some essential properties. The `System.getConfig()` looks like this
```javascript
{
  "paths": Object,
  "meta": Object,
  "baseURL": "http://localhost:3000/",
  "map": Object,
  "warnings": false,
  "defaultJSExtensions": false,
  "pluginFirst": false,
  "loaderErrorStack": false,
  "packages": Object,
  "packageConfigPaths": Array,
  "bundles": Object,
  "depCache": Object,
  "production": false
}
```
Even though [this](https://github.com/casperlamboo/plugin-worker/blob/master/jspm.config.js) is the `config.jspm.js`. Some properties defined in `config.jspm.js` are missing, such as
 - browserConfig
 - devConfig
 - transpiler

I believe `browserConfig` and `devConfig` are not that necessary but I could really use the `transpiler` property.

**Comment by guybedford** [comment]
Thanks, I've fixed this in https://github.com/systemjs/systemjs/commit/3341825e6656c2bd31e8017595e783eec4905970.

**Comment by guybedford** [comment]
Released in 0.19.34.
